environment.yml

```diff
@@ -11,7 +11,7 @@ dependencies:
     - pandas=0.23.0
     - pytz=2017.3
     - pip:
-        - allennlp==0.8.1
+        - allennlp==0.8.4
```

Comment:
For compatibility with the latest pytorth_pretrained_bert. Doesn't break anything else.

Reply:
False alarm. This does break stuff.

Reply:
Has something been done to make sure this does not break things?